app/src/middleware.ts

```diff
@@ -11,7 +11,7 @@ import { defaultLocale, locales } from "./i18n/config";
 
 // Don't run middleware on API routes or Next.js build output
 export const config = {
-  matcher: ["/((?!api|_next|.*\\..*).*)"],
+  matcher: ["/((?!api|_next|sitemap|.*\\..*).*)"],
```

Comment:
Allows locahost:3000/sitemap.xml to resolve (I'm not entirely sure the i18n reason why) - we had to do this on grants